aten/src/ATen/native/quantized/cpu/ACLUtils.h

```diff
@@ -0,0 +1,251 @@
+#pragma once
+
+#include <ATen/Config.h>
+#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
+
+#include <ATen/native/quantized/cpu/OnednnUtils.h>
+#include <arm_compute/core/Error.h>
+#include <arm_compute/core/TensorInfo.h>
+#include <arm_compute/function_info/ActivationLayerInfo.h>
+#include <arm_compute/runtime/Allocator.h>
+#include <arm_compute/runtime/NEON/functions/NEActivationLayer.h>
+#include <arm_compute/runtime/NEON/functions/NEGEMMLowpMatrixMultiplyCore.h>
+#include <arm_compute/runtime/NEON/functions/NEQuantizationLayer.h>
+#include <arm_compute/runtime/Tensor.h>
+#include <array>
+
```

Comment:
Prefer placing nonmember functions in a namespace; use completely global functions rarely.
```suggestion
namespace at::native {
```

Reply:
I placed the "helper" structs (e.g. `ACLDynamicQuantMatmul`) inside an `acl_utils` namespace. 
Does it look better now?